common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -236,10 +236,13 @@ object ContentWidths {
 
   object GalleryMedia {
     val inline = WidthsByBreakpoint(
-      mobile          = Some(445.px),
-      mobileLandscape = Some(610.px),
-      phablet =         Some(620.px),
-      tablet =          Some(700.px)) // desktop, leftCol, and wide are also 700px
+      mobile          = Some(480.px),
+      mobileLandscape = Some(660.px),
+      phablet =         Some(700.px),
+      tablet =          Some(700.px), // TODO: Change to 480 when new galleries is merged
+      desktop =         Some(720.px),
+      leftCol =         Some(880.px),
+      wide =            Some(1010.px))
```

Comment:
Unnecessarily downloading lower resolution sources from the set when the viewport shrinks (instead of just shrinking already downloaded ones) looks like a bug (or bad policy decision) on Firefox’s part. All other browsers I tested do not trigger extra downloads (test page https://samefourchords.com/2014/11/01/zakopane-poland):

## Chrome 51:
![chrome 51](https://cloud.githubusercontent.com/assets/6032869/15099568/b353d88c-1550-11e6-847c-2883e8e7f803.gif)
---
## Firefox 47 (same on 49 :sob:):
![firefox 47](https://cloud.githubusercontent.com/assets/6032869/15099570/bf332996-1550-11e6-84ad-965e6a7a54f4.gif)
---
I couldn’t find a related bugzilla entry or anything at all TBH, just someone having the same problem: https://www.sitepoint.com/community/t/img-srcset-firefox-loads-too-many-images/219162...

One could argue that lower resolution images may look better at native res not having to undergo resizing and not exhibiting related interpolation artifacts, but one would be wrong, of course.